### ISSUE TITLE
Angular 4 - Unable to set size

### ISSUE THREAD

**Issue by albertStaalburger** [optimistex/ngx-select-ex#34]
I am trying to set the `size` property but it is not being applied.
Here is the select I am using:
```
<ngx-select [formControl]="ngxControl"
        [allowClear]="true"
        [items]="this.departureCities" 
        optionValueField="townId" optionTextField="townDesc"
        size="large"
        (select)="departCityChange($event)"
        [(ngModel)]="departCityId"
        placeholder="No departure city selected">
</ngx-select>
```

I tried setting it globally but got the following errors:
`"...node_modules/ngx-select-ex/ngx-select-ex"' has no exported member 'INgxSelectOptions'.`
`Property 'forRoot' does not exist on type 'typeof NgxSelectModule'.`

My `app.module`
```
import { NgxSelectModule, INgxSelectOptions } from 'ngx-select-ex';
...

const CustomSelectOptions: INgxSelectOptions = {
    size: 'large'
};

@NgModule({
        imports: [
                BrowserModule,
                NgxSelectModule.forRoot(CustomSelectOptions),
                ...
        ],
```

**Comment by optimistex** [comment]
Hm... Yes... I do not always update version for Angular4 with newest features.
Wait. I will release version for Angular4...

**Comment by albertStaalburger** [comment]
Ok, so this feature is only available in Angular < 4?

**Comment by optimistex** [comment]
> Ok, so this feature is only available in Angular < 4?

The trouble in another one. For building the package under Angular4 I have to make an environment of Angular4 because package for Angular5 doesn't work with Angular4.

**Comment by albertStaalburger** [comment]
I understand.

I will try and find a work around in the meantime.

Thank you.

**Comment by optimistex** [comment]
Published: ngx-select-ex@3.4.0-ng4

Please install and let me know if something wrong:
`npm i ngx-select-ex@ng4 --save`

**Comment by albertStaalburger** [comment]
It is working.
Thank you.
